snap/snapcraft.yaml

```diff
@@ -22,49 +22,33 @@ apps:
       CLASSPATH: "$($HADOOP_HOME/bin/hdfs classpath --glob)"
       PATH: "/snap/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:$SNAP/usr/bin:$SNAP/bin:$HADOOP_HOME/sbin:$HADOOP_HOME/bin:$PATH"
 parts:
-  dvc-libs:
-    plugin: nil
+  dvc:
+    plugin: python
+    source: https://github.com/iterative/dvc.git
+    source-tag: 3.44.0
     build-packages:
-    - curl
-    - tar
-    - gzip
+    - git
+    - libc-dev
     stage-packages:
     - git
     - libpython3-stdlib
-    - libpython3.8-stdlib
-    - libpython3.8-minimal
+    - libpython3.10-stdlib
+    - libpython3.10-minimal
     - python3-pip
     - python3-setuptools
     - python3-wheel
     - python3-venv
     - python3-minimal
     - python3-distutils
     - python3-pkg-resources
-    - python3.8-minimal
-    override-build: |
-        snapcraftctl build
-        curl -sSLO https://s3-us-east-2.amazonaws.com/dvc-public/dvc-test/openjdk-7u75-b13-linux-x64-18_dec_2014.tar.gz
-        curl -sSLO https://s3-us-east-2.amazonaws.com/dvc-public/dvc-test/hadoop-2.7.2.tar.gz
-        tar -xzf openjdk-7u75-b13-linux-x64-18_dec_2014.tar.gz -C $SNAPCRAFT_PART_INSTALL
-        tar -xzf hadoop-2.7.2.tar.gz -C $SNAPCRAFT_PART_INSTALL
```

Comment:
Not sure if this even worked before. We didn't even `make` it before. I'll run with what we have right now to save time.